src/main/java/hudson/plugins/jira/JiraBuildAction.java

```diff
@@ -11,11 +15,12 @@
  * @author Kohsuke Kawaguchi
  */
 public class JiraBuildAction implements Action {
-    public final AbstractBuild<?, ?> owner;
+    
+	public final Run<?, ?> owner;
```

Comment:
Binary compatibility also includes API.
If somebody uses <code>AbstractBuils<?,?> build = x.getAction(JiraBuildAction.class)owner</code> somewhere in the dependent code, this method will become non-compilable and cause exceptions in Runtume